terraform/environments/mlra/ecs.tf

```diff
@@ -33,79 +33,10 @@ module "mlra-ecs" {
   ecs_scaling_cpu_threshold = local.application_data.accounts[local.environment].ecs_scaling_cpu_threshold
   ecs_scaling_mem_threshold = local.application_data.accounts[local.environment].ecs_scaling_mem_threshold
 
-  depends_on = [module.alb, aws_cloudwatch_log_group.ecs_log_group] # TODO module.alb dependancy may have to be re-factored further into development
```

Comment:
Removing dependency on the entire ALB module to minimise changes of TF unnecessarily recreating unaffected resources